integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/glue/steps/database/ws/DistributionAutomationNotificationWebServiceConfigurationRepository.java

```diff
@@ -9,16 +9,11 @@
  */
 package org.opensmartgridplatform.cucumber.platform.distributionautomation.glue.steps.database.ws;
 
-import java.util.List;
 import org.opensmartgridplatform.adapter.ws.domain.entities.ApplicationDataLookupKey;
 import org.opensmartgridplatform.adapter.ws.domain.entities.NotificationWebServiceConfiguration;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface DistributionAutomationNotificationWebServiceConfigurationRepository
-    extends JpaRepository<NotificationWebServiceConfiguration, ApplicationDataLookupKey> {
-
-  List<NotificationWebServiceConfiguration> findByIdOrganisationIdentification(
-      String organisationIdentification);
-}
+    extends JpaRepository<NotificationWebServiceConfiguration, ApplicationDataLookupKey> {}
```

Comment:
This could probably also become:
```suggestion
    extends NotificationWebServiceConfigurationRepository {}
```